tests/subworkflows/test_lint.py

```diff
@@ -434,3 +435,60 @@ def test_subworkflows_lint_local_old_format(self):
         assert len(subworkflow_lint.failed) == 0, f"Linting failed with {[x.__dict__ for x in subworkflow_lint.failed]}"
         assert len(subworkflow_lint.passed) > 0
         assert len(subworkflow_lint.warned) >= 0
+
+
+class TestSubworkflowsLintPatch(TestSubworkflows):
+    def setup_patch(self, pipeline_dir):
```

Comment:
I think this can be `setUp` so it is automatically called for each test in this class